plutus-benchmark/bls-benchmarks/exe/Main.hs

```diff
@@ -0,0 +1,161 @@
+-- editorconfig-checker-disable-file
+{-# LANGUAGE NumericUnderscores #-}
+{-# OPTIONS_GHC -Wno-unused-top-binds #-}
+{-# OPTIONS_GHC -Wno-name-shadowing #-}
+
+{- | Print out the costs of various test scripts involving the BLS12_381
```

Comment:
Right, so I think what I'm suggesting is basically to make this into a test suite with budget tests instead. Maybe the output is less easy to read, IDK.

Reply:
We could also check in the output as a golden file.